submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,367 @@
+
+## Making the most of your uranium
+
+Despite its massive energy gains and the resulting coal and oil savings, nuclear fuel might not be perfect. The uranium-235 that it uses could instead be used to make uranium fuel cells, which provide a different energy situation: They can only be used inside nuclear reactors. One fuel cell provides one reactor a constant supply of 40MW of heat for exactly 200 seconds, which adds up to a total energy of `40MW * 200s = 8000MJ = 8GJ`. If the reactor has any neighbors, the neighbor bonus kicks in, and the bonus from each neighbor leads to an additional 40MW of heat being supplied by the same fuel cell. Since the reactor can up to 4 neighbors, the fuel cell can provide up to 5 times its original yield, which is `8GJ * 5 = 40GJ`. We already dealing with GJ instead of MJ when discussing a single fuel cell but the recipe for them yields 10 per piece of uranium-235:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing uranium-235 into uranium fuel cells'
+ caption='Processing uranium-235 into uranium fuel cells'
+
+%}
+
+Hence, we see that we can get up to hundreds of times more energy from the same piece of uranium-235 if it is used in a nuclear reactor instead of nuclear fuel. This would suggest that if one has a uranium-235 supply already, they should just switch to nuclear power and electric machines instead of burner devices. However, it is also worth noting that nuclear fuel is unlocked alongside Kovarex Enrichment, which allows for the easy production of uranium-235. This means that using uranium efficiently stops being an issue around the time you unlock nuclear fuel.
+
+## Module effects on energy gains
+
+So far we have done our calculations without using any modules for the sake of keeping things simple and understanding the default conditions. Now let's look at them again with modules and see if our results change significantly. Modules primarily affect the power usage of the fuel processing machines. Let's first review the equation:
+
+`Total processing energy (E) = Machine power usage (P) * time of 1 crafting cycle (t_c) * numnber of crafting cycles (n) / crafting speed (C)`
+
+`E = P * t_c * n / C`
+
+Here is what the modules do:
+
+- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules because they are quite cheap to make and they cut down power usage by 60% or even 80% (the efficiency limit), depending on how many module slots the machines have.
```

Comment:
```suggestion
- **Efficiency modules:** They reduce the power usage of the fuel processing machines ("P"). Their overall effects on energy gains are small but noteworthy for most recipes. I would definitely recommend using efficiency 1 modules in all of the fuel processing machines if you don't use any other modules, because they are quite cheap to make and they cut down power usage by 60% or even 80% (the efficiency limit), depending on how many module slots the machines have.
```